redbot/core/commands/converters.py

```diff
@@ -0,0 +1,80 @@
+import re
+from datetime import timedelta
+from typing import TYPE_CHECKING
+
+from redbot.core.commands import Converter, BadArgument
+
+if TYPE_CHECKING:
+    from .commands import Context
+
+__all__ = ["TimedeltaConverter"]
+
+TIME_RE = re.compile(
+    r"((?P<days>\d+?)\s?(d(ays?)?))?\s?((?P<hours>\d+?)\s?(hours?|hrs|hr?))?\s?((?P<minutes>\d+?)\s?(minutes?|mins?|m))?\s?((?P<seconds>\d+?)\s?(seconds?|secs?|s))?\s?",
+    re.I,
+)
+
+
+class TimedeltaConverter(Converter):
+    """
+    Convert to a :class:`datetime.timedelta` class.
+
+    The converter supports seconds, minutes, hours and days.
+
+    Time can be attached to the unit (only use the first letter) or not
+    (then use the full word).
+
+    .. admonition:: Example
+
+        *   ``50s`` = ``50 seconds`` -> :class:`datetime.timedelta(seconds=50)`
+        *   ``12m`` = ``12 minutes`` -> :class:`datetime.timedelta(seconds=720)`
+        *   ``4h`` = ``4 hours`` -> :class:`datetime.timedelta(seconds=14400)`
+        *   ``1d`` = ``1 day`` -> :class:`datetime.timedelta(days=1)`
+
+        Using the converter with a command:
+
+        .. code-block:: python3
+
+            @commands.command()
+            async def timer(self, ctx, time: commands.Timedelta):
+                await self.start_timer(time)
+
+        Using the converter manually:
+
+        .. code-block:: python3
+
+            async def convert_time(ctx: Context, text: str) -> datetime.timedelta:
+                time = await commands.Timedelta.convert(ctx, text)
+                return time
+    """
```

Comment:
Couple issues with these code blocks.
1. lines 39 and 47: it's TimedeltaConverter, not Timedelta.
2. line 48: You're using the class to call an instance's method. You need to use an instance to call an instance method.